Documentation/operations/upgrade.rst

```diff
@@ -406,6 +406,12 @@ Deprecated Options
 * The ``cluster-pool-v2beta`` IPAM mode is deprecated and will be removed in v1.15.
   The functionality to dynamically allocate Pod CIDRs is now provided  by the
   more flexible ``multi-pool`` IPAM mode.
+* The ``kube-proxy-replacement`` option's values ``strict``, ``partial`` and
+  ``disabled`` are deprecated and will be removed in v1.15. They are replaced
```

Comment:
I thought we agreed to not remove them and let them remain as hidden alias?

Reply:
Yep, we will (for those who don't read upgrade notes). I added the note about removal to motivate users to migrate to new values.

Reply:
Hehe, ok then.